working_groups/wg-Cloud/README.md

```diff
@@ -22,9 +23,5 @@
 * [Google Group](https://groups.google.com/forum/#!forum/salt-cloud)
 
 ## Meeting Details
-* [2019-June-14](meeting-notes/2019-06-14.md) Meeting Notes
```

Comment:
@dwoz Since the meeting notes were directly added to README file, repo doesn't have those changes. I will wait for your comments before merging this PR changes